pkg/detectors/typeform/v2/typeform_integration_test.go

```diff
@@ -1,6 +1,3 @@
-//go:build detectors
```

Comment:
question: Have we started removing the build tag from detector integration tests, or is it only being removed from non-integration tests? Sorry if i'm late to the part on this. 😅

Reply:
Ah good catch 😃 I removed it accidentally